person.rb

```diff
@@ -24,14 +31,9 @@ def of_age?
   end
 end
 
-j = Person.new(18, 'John')
-pp j
-j.can_use_services?
-
-m = Person.new(16, 'Martin', parent_permission: false)
-pp m
-m.can_use_services?
-
-paul = Person.new(16, 'Paul')
-pp paul
-paul.can_use_services?
+person = Person.new(22, 'maximilianus')
+person.correct_name
+capitalized_person = CapitalizeDecorator.new(person)
+capitalized_person.correct_name
+capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
+capitalized_trimmed_person.correct_name
```

Comment:
- [ ] Running this file gives no output, kindly fix this issue so that you are able to decorate a class as this is part of the requirements.
![image](https://user-images.githubusercontent.com/81932478/215620288-22dd54c3-493e-4608-b100-295eb733668e.png)

Reply:
well resolved! ✔️